.npmrc

```diff
@@ -1 +1,2 @@
-legacy-peer-deps=true
+strict-peer-dependencies=false
+auto-install-peers=false
```

Comment:
Please add newline at the end

Reply:
I followed the official pnpm documentation [here](https://pnpm.io/npmrc#peer-dependency-settings). My understanding says that `strict-peer-dependencies` and  `auto-install-peers` are the replacement of `legacy-peers-deps` flag

Reply:
Sorry, the comment is related to the newline character at the end of the file. GitHub shows a big red symbol in the code preview then the newline character is missing, which is ugly

![Screenshot 2022-09-07 at 12 36 05 PM](https://user-images.githubusercontent.com/74390/188845621-90fdb0e1-345c-4d51-9eaa-3283ea4ed6fa.png)